trendico/models.py

```diff
@@ -1,3 +1,48 @@
 from django.db import models
-
+from datetime import datetime, timedelta
 # Create your models here.
+CATEGORY_CHOICES = (
+    ('L', 'Laptop'),
+    ('T', 'Tab'),
+    ('C', 'Camera'),
+    ('H', 'Headphone')
+)
+class Product(models.Model):
+    product_name=models.CharField(max_length=50)
+    img_URL = models.ImageField(upload_to='images')
+    price = models.FloatField()
+    discount_price = models.FloatField(blank=True, null=True)
+    category = models.CharField(choices=CATEGORY_CHOICES, max_length=1)
+    description = models.TextField()
+    total_quantity=models.PositiveIntegerField()
```

Comment:
using this way of stock, we wouldn't be able to know that how many quantity was at start and how much is remaining? make a flow to handle product stock where we can know about the quantity at start and how is remaining, even we can show an alert when the remaining items are less than 5